docs/docs/getting-started/installation/aws.md

```diff
@@ -0,0 +1,163 @@
+---
+title: AWS
+sidebar_label: AWS
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import TLDR from "@site/src/components/TLDR";
+import SuccessBox from "@site/src/components/SuccessBox";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<TLDR>
+
+Install Airy Core on AWS with one command.
+
+</TLDR>
+
+The goal of this document is to provide an overview of how to run Airy Core on
+AWS cloud platform, using the [AWS Elastic Kubernetes Service](https://aws.amazon.com/eks/).
+
+Apart from the EKS cluster, the installation will create all the necessary AWS resources, such as:
+
+- VPC resources (VPC, subnets, route tables, public gateways)
+- IAM roles and policy attachments
+- EKS cluster and EKS node groups
+- EC2 instances, as part of the created node group
+
+## Creating a cluster
+
+For creating the cluster you would need to setup your local AWS environment, by [configuring your local AWS profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html) for the AWS account where all the resources will be created.
+
+Download and install the [Airy CLI](cli/installation.md).
+
+Export your AWS_PROFILE as described in the [AWS documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html).
+
+Now you can run this command, which will create Airy core in your AWS account:
+
+```bash
+airy create --provider=aws
+```
+
+By default, the installation will create a single EC2 Kubernetes node, as part of a single node group. You can scale your EKS cluster by adding more nodes or node groups, through the AWS web console or the AWS CLI.
+
+This will print URLs for accessing the UIs and APIs as seen in this recording:
+
+import Script from "@site/src/components/Script";
+
+<Script data-cols="90" id="asciicast-KHz6yTASgqwnKT5FB7Gt7jF1k" src="https://asciinema.org/a/KHz6yTASgqwnKT5FB7Gt7jF1k.js"></Script>
+
+If you want to customize your `Airy Core` instance please see our [Configuration Section](configuration.md).
+
+## Public webhooks
+
+The public webhooks will be accessible on the public LoadBalancer which is created by the Ingress loadBalancer Kubernetes service.
+
+To get the public URL of your AWS Airy Core installation you can run:
+
+```sh
+kubectl --kubeconfig ~/.airy/kube.conf get --namespace kube-system service traefik --output jsonpath='{.status.loadBalancer.ingress[0].hostname}'
+```
+
+## Where to go from here
+
+Now that you have a running local installation of Airy core on AWS you can connect it to messaging sources. Check out the
+[source documentation](/sources/introduction.md) to learn more.
+
+## Third party tools
+
+Third party tools can be activated in the `airy.yaml` configuration file, under the `tools` section.
+For more details please see our [Configuration Section](configuration.md).
+
```

Comment:
For the Uninstall part, it can be also a script which gets the id and removes all the resources.
However, for the first version, I thought better to have separate steps, as sometimes something fails to delete and it is not fully stable yet. When something fails, it should be deleted from the AWS console.